cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/selector/BaseStorageSelector.java

```diff
@@ -0,0 +1,15 @@
+package com.linkedin.openhouse.cluster.storage.selector;
+
+import com.linkedin.openhouse.cluster.storage.configs.StorageProperties;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/** An abstract class for all Storage selectors */
+public abstract class BaseStorageSelector implements StorageSelector {
+
+  @Autowired private StorageProperties storageProperties;
```

Comment:
Lets use the StorageProperties for core logic inside `DefaultStorageSelector` or `StorageInNameSelector`. If we use the name in storageProperties, the behavior will be incorrect.